### PR TITLE
A couple more testsets for operators

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]

File: test/operators.jl [hunk 1]
```diff
@@ -2,54 +2,58 @@
 
 using Random: randstring
 
-@test ifelse(true, 1, 2) == 1
-@test ifelse(false, 1, 2) == 2
+@testset "ifself" begin
```
Comment (JeffBezanson) [inline test/operators.jl]:
```suggestion
@testset "ifelse" begin
```

Comment (kshyatt) [reply test/operators.jl]: Whoops!

**Comment by fredrikekre** [comment]
I am sure it passes, but maybe should have run CI?

**Comment by kshyatt** [comment]
Oh yuck, it did run the first time and I thought it had again. Sorry!

**Pull request merged**